src/bin/stratisd.rs

```diff
@@ -355,16 +355,14 @@ fn run(matches: &ArgMatches, buff_log: &buff_log::Handle<env_logger::Logger>) ->
 
     0   == Always udev fd index
     1   == SIGNAL FD index
-    2   == TIMER FD for periodic dump index
-    3   == engine index if eventable
-    3/4 == Start of dbus client file descriptor(s)
-            * 3 if engine is not eventable
-            * else 4
+    2   == engine index if eventable
+    2/3 == Start of dbus client file descriptor(s)
+            * 2 if engine is not eventable
+            * else 3
```

Comment:
I'm a little confused by the documentation here. This seems like it was here before you touched it, but from my reading, the first line seems to indicate that 2 is eventable and the second bullet point says 2 if not eventable. @mulkieran can you potentially shed a little light on what this might be trying to communicate?

Reply:
> I'm a little confused by the documentation here. This seems like it was here before you touched it, but from my reading, the first line seems to indicate that 2 is eventable and the second bullet point says 2 if not eventable. @mulkieran can you potentially shed a little light on what this might be trying to communicate?

The dbus index start gets shifted left to 2 if the engine is not eventable; the sim engine actually isn't eventable.

Reply:
Oh okay! It was a little bit hard for me to parse at first, but this makes a lot more sense.